diamm/serializers/website/stats.py

```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```

Comment:
Or are you suggesting that I rename the keys in the view before I pass them to the serializer?